packages/front/src/views/Map.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="container">
+    <div class="search-wrapper">
+      <SearchInput />
+    </div>
+    <img src="../assets/images/map_placeholder.png" alt="placeholder" />
+  </div>
+  <Footer />
```

Comment:
Use Element-plus component to define layout
https://element-plus.org/en-US/component/container.html

You can define height of footer as eg `48px` and height of the main component as `calc(100vh - 48px)`